sys/include/color.h

```diff
@@ -104,6 +105,22 @@ void color_str2rgb(const char *str, color_rgb_t *color);
  */
 void color_rgb2str(const color_rgb_t *rgb, char *str);
 
+/**
+ * @brief Invert a given rgb color
+ *
+ * @param[in] rgb       Input rgb color, that should be converted.
+ * @param[out] inv_rgb  Output rgb color, result of the conversion.
+ */
+void color_rgb_invert(const color_rgb_t *rgb, color_rgb_t *inv_rgb);
+
+/**
+ * @brief Calculate the complementary color of a given rgb color
+ *
+ * @param[in] rgb       Input rgb color.
+ * @param[out] comp_rgb Output rgb color, result of the complementary color calculation.
+ */
+void color_rgb_complementary(const color_rgb_t *rgb, color_rgb_t *comp_rgb)
```

Comment:
Could you add that link, or at least refer to it as Adobe Complementary or something like that in the doc of this function then, please?